src/utils/src/lib.rs

```diff
@@ -1,16 +1,14 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#[macro_use]
 extern crate bitflags;
```

Comment:
`bitflags` was used in `utils` only for `epoll`. It can be removed now (and from utils `Cargo.toml` too). Thanks!